codalab/rest/schemas.py

```diff
@@ -124,6 +124,8 @@ class WorksheetSchema(Schema):
     title = fields.String()
     frozen = fields.DateTime(allow_none=True)
     is_anonymous = fields.Bool()
+    date_created = fields.DateTime()
```

Comment:
Should we do this too?:

```suggestion
    date_created = fields.DateTime(allow_none=True)
```

Reply:
good catch